### PR TITLE
build: Cleanup Dependencies

### PR DESCRIPTION
**Pull request by dwijnand**
(I was reviewing how addSbtModule was setup, and cleaned the code up to read it.)

### CODE REVIEW

File: project/Dependencies.scala [hunk 1]
```diff
@@ -6,121 +6,97 @@ object Dependencies {
...
-  private val compilerApiInfo = "org.scala-sbt" %% "zinc-apiinfo" % zincVersion
-  private val compilerBridge = "org.scala-sbt" %% "compiler-bridge" % zincVersion
-  private val zinc = "org.scala-sbt" %% "zinc" % zincVersion
-  private val zincCompile = "org.scala-sbt" %% "zinc-compile" % zincVersion
-  private val zincCompileCore = "org.scala-sbt" %% "zinc-compile-core" % zincVersion
-
-  def getSbtModulePath(key: String, name: String) = {
+  private val libraryManagementIvy  = "org.scala-sbt" %% "librarymanagement-ivy"  % lmVersion
+
+  val launcherVersion   = "1.1.3"
```
Comment (eed3si9n) [inline project/Dependencies.scala]: Is this vertically aligned? I think we use Scalafmt to format these files and I don't think this would stick.

Comment (dwijnand) [reply project/Dependencies.scala]: 💩

**Comment by dwijnand** [comment]
Apparently Travis CI creates PR-specific caches, so somehow the cache for this PR had become corrupt which kept killing the sbt launching.  Finally, it's green after deleting that PR-specific cache.

**Comment by typesafe-tools** [comment]
A validation involving this pull request <a href="https://jenkins.scala-sbt.org:8592/job/sbt-validator/1924/console">is in progress...</a>

**Comment by typesafe-tools** [comment]
[//]: # (Refs: pull/5358/head-develop-develop-develop-develop)
The validator <A HREF="https://jenkins.scala-sbt.org:8592/job/sbt-validator/1924/">has checked</A> the following projects, tested using sources.sh.

Project             | Reference            | Commit
:------------------ | :------------------- | :----------------
  sbt               | <A HREF="https://github.com/sbt/sbt/pull/5358">pull/5358/head      </A> | sbt/sbt@6b7b572e1a5dd52bb67c4461bcc5c3af8c2a21df
  zinc              | <A HREF="https://github.com/sbt/zinc/tree/develop">develop             </A> | sbt/zinc@93fdc353140627fb37feb227294c47e645d04202
  io                | <A HREF="https://github.com/sbt/io/tree/develop">develop             </A> | sbt/io@d3297b56297a524baf5982661170edfdb8a6823e
  librarymanagement | <A HREF="https://github.com/sbt/librarymanagement/tree/develop">develop             </A> | sbt/librarymanagement@3132a3eb59fcfbabf33683388377b39a5458457d
  util              | <A HREF="https://github.com/sbt/util/tree/develop">develop             </A> | sbt/util@098796cdff7305a4aaadea9fdbba7fb6c23ff3ed

:x: The result is: ***FAILED***
<A HREF="https://jenkins.scala-sbt.org:8592/job/sbt-validator/parambuild?sbt=pull/5358/head&zinc=develop&io=develop&librarymanagement=develop&util=develop&deploy_to_bintray=false&run_sbt_community=false">(restart)</A>

**Review by eed3si9n — APPROVED** [review]
Looks like scripted mostly ran, so LGTM

**Pull request merged**